src/react/projects/spark-core-react/src/SprkFooter/components/SprkFooterConnectIcons/SprkFooterConnectIcons.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import { uniqueId } from 'lodash';
+import PropTypes from 'prop-types';
+import SprkIcon from '../../../SprkIcon/SprkIcon';
+
+class SprkFooterConnectIcons extends Component {
+  constructor(props) {
+    super(props);
+
+    if (props.connectIcons.icons !== undefined) {
+      this.state = {
+        connectIconsHasIds: props.connectIcons.icons.map(
+          icon => ({ id: uniqueId(), ...icon }),
+        ),
+      };
+    }
+  }
+
+  render() {
+    const { connectIcons } = this.props;
+    if (connectIcons.icons === undefined) return false;
```

Comment:
this check shouldn't be necessary, if its required, it cant be undefined, if its not it should have a default value